js/coinsbit.js

```diff
@@ -0,0 +1,1103 @@
+                    'public': 'https://coinsbit.io',
+                    'private': 'https://coinsbit.io',
+                },
+                'www': 'https://coinsbit.io/',
+                'doc': [
+                    'https://www.notion.so/coinsbitwsapi/API-COINSBIT-WS-API-COINSBIT-cf1044cff30646d49a0bab0e28f27a87',
+                    'https://github.com/Coinsbit-connect/api/wiki/API',
+                ],
+                'fees': [
+                    'https://coinsbit.io/fee-schedule',
+                ],
+                'referral': '  <<<TODO>>>   ',
+            },
+            'api': {
+                'public': {
+                    'get': {
+                        'markets': 1,
+                        'tickers': 1,
+                        'ticker': 1,
+                        'book': 1,
+                        'history': 1,
+                        'history/result': 1,
+                        'products': 1, // Of no use, everything already available in 'markets'
+                        'symbols': 1, // Of no use, everything already available in 'markets'
+                        'depth/result': 1,
+                        'kline': 1,
+                    },
+                },
+                'private': {
+                    'post': {
+                        'order/new': 1,
+                        'order/new_market': 1,
+                        'order/cancel': 1,
+                        'orders': 1,
+                        'account/balances': 1,
+                        'account/balance': 1,
+                        'account/order': 1, // theoretically, this might got implemented in fetchClosedOrder, but API is broken
+                        'account/trades': 1,
+                        'account/order_history': 1,
+                        'account/order_history_list': 1,
+                    },
+                },
+            },
+            'fees': {
+                'trading': {
+                    'tierBased': false,
+                    'percentage': true,
+                    'maker': 0.2 / 100,
+                    'taker': 0.2 / 100,
+                },
+            },
+            'options': {
+                'createMarketBuyOrderRequiresPrice': true,
+                'fetchTradesMethod': 'publicGetHistory', // publicGetHistory | publicGetHistoryResult
+                'maxDiapasonsForTimeframes': {
+                    '1m': 43200,
+                    '30m': 86400,
+                    '1h': 86400 * 7,
+                    '1d': Number.MAX_SAFE_INTEGER,
+                },
+            },
+            'commonCurrencies': {
+                // 'xxx': 'xxxxxxx',
+            },
+            'exceptions': {
+                'exact': {
+                },
+                'broad': {
+                },
+            },
+        });
+    }
+
+    async fetchMarkets (params = {}) {
+        let data = undefined;
+        const request = params;
+        const response = await this.publicGetMarkets (request);
```

Comment:
The `request` variable is not necessary here.
We can also create ` const data = this.safeValue (response, 'result', [])` after receiving the response.